dot/core/messages_test.go

```diff
@@ -157,8 +157,9 @@ func TestService_HandleTransactionMessage(t *testing.T) {
 	extBytes := createExtrinsics(t, s.rt, genHash, 0)
 
 	msg := &network.TransactionMessage{Extrinsics: []types.Extrinsic{extBytes}}
-	err = s.HandleTransactionMessage(msg)
+	b, err := s.HandleTransactionMessage(msg)
```

Comment:
can you also add a test case for invalid extrinsics?

Reply:
added.